Pinta.Core/Extensions/CairoExtensions.cs

```diff
@@ -873,33 +895,26 @@ public static ColorBgra GetBilinearSampleWrapped (
 				uint wll = sxfracinv * syfrac;
 				uint wlr = sxfrac * syfrac;
 
-				int sx = iu;
-				if (sx < 0)
-					sx = srcWidth - 1 + ((sx + 1) % srcWidth);
-				else if (sx > (srcWidth - 1))
-					sx %= srcWidth;
+				int sx =
```

Comment:
IMO having nested ternary operators like this is less readable than the original code.

What about just doing something like the following to eliminate re-assignment?
```csharp
int sx;
if (sx < 0)
    sx = ...;
else if (sx > (srcWidth - 1))
    sx = ...;
else
    sx = iu;
```

Reply:
I just implemented your suggestion @cameronwhite